Firestore/core/src/local/index_manager.h

```diff
@@ -58,6 +71,65 @@ class IndexManager {
    */
   virtual std::vector<model::ResourcePath> GetCollectionParents(
       const std::string& collection_id) = 0;
+
+  /**
+   * Adds a field path index.
+   *
+   * The actual entries for this index will be created and persisted
+   * in the background by the SDK, and the index will be used for query
+   * execution once values are persisted.
```

Comment:
NIt: This looks like it is not using the full line length.

Reply:
Fixed.